2021-10-13-github-templates.md

```diff
@@ -0,0 +1,123 @@
+# OmbuLabs Standard Github Templates
+
+## Problem:
+
+New and junior contributors might be unsure about the information we require to
+efficiently review a PR. Similarly, it might be unclear what information is required when
+a new BUG or FEATURE is logged as an issue in Github.
+
+We have existing PR and ISSUE templates but they are inconsistent and might ask for
+unnecessary information from the contributor.
+
+The goal is to simplify  and streamline the steps required to make a contribution or log
+an issue.
+
+## Proposal:
+
+Come up with a consistent set of Github templates. The templates should include prompts
+only the ESSENTIAL information we require. We want to make sure that we don't ask for more
+information than we require - as this will discourages contributions.
+
+## Templates:
+
+### Pull Requests
+
+```
+    <!--- Before we start... -->
+    <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
```

Comment:
If there's a possibility they feel unsure, probably we are asking too many questions

Reply:
I still think this is unclear for the contibutor, ask were? create an issue? some other place? this should be clear but there should be a `how to contribute` section in the project and not here in the template, maybe the template can link to that document

Reply:
Agree on linking to the contributing guide, a-lot of what we are putting in this template should be there, the template should be smaller :squirrel: